packages/features/ee/workflows/lib/test/workflows.test.ts

```diff
@@ -96,17 +96,17 @@ const mockBookings = [
     eventTypeId: 1,
     userId: 101,
     status: BookingStatus.ACCEPTED,
-    startTime: `2024-05-20T09:00:00.000Z`,
-    endTime: `2024-05-20T09:15:00.000Z`,
+    startTime: `2024-05-21T09:00:00.000Z`,
```

Comment:
changing the date of mock bookings to avoid having past reminders in existing tests